src/main/webapp/2DVisual.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Simple Map</title>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDCgKca9sLuoQ9xQDfHUvZf1_KAv06SoTU&callback=initMap&libraries=&v=weekly"
```

Comment:
api key protection?

Reply:
I talked to Jared about this earlier, he said as long as we restrict the key to be used by just our domain we're okay to push the Maps API key.